exercises/practice/phone-number/.meta/PhoneNumber.example.ps1

```diff
@@ -0,0 +1,70 @@
+Function Get-PhoneNumber() {
+    <#
+    .SYNOPSIS
+    Clean up user-entered phone numbers so that they can be sent SMS messages.
+
+    .DESCRIPTION
+    Given a phone number string, check if it's a valid phone number that complied with the NANP system.
+    Return the cleaned number string if it's valid, otherwise throw the relevant error.
+    Also provide user the option to print out the number in pretty format.
+
+    .PARAMETER Number
+    The phone number string to be processed.
+
+    .PARAMETER Pretty
```

Comment:
These option isn't part of the problem spec, is it borrowed from another track?

Reply:
Yeah, i took inspiration from the python track, saw it as a nice opportunity to introduce the switch parameter, make the function more like a real command line when we add extra flag to modify behavior.

Reply:
Yeah, but it would be good if you write where the tests have been taken from, when you open a pr. Unless the tests are taken from problem spec: https://github.com/exercism/problem-specifications/tree/main/exercises